heroic-component/src/main/java/com/spotify/heroic/aggregation/BucketAggregationInstance.java

```diff
@@ -267,4 +252,13 @@ public String toString() {
     private interface BucketConsumer<B extends Bucket, M extends Metric> {
         void apply(B bucket, M metric);
     }
+
+    /**
+     * A start, and an end bucket (exclusive) selected.
```

Comment:
Why exclusive here? 
I'm thinking that it would be more intuitive to simply specify the indices of the first and last (inclusive) buckets in the wanted bucket range - The bucket range that will include all samples for the wanted time range [start, end)